src/Chapter1.hs

```diff
@@ -653,7 +674,12 @@ You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
 
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n =
+  if div (abs n) 10  == 0 then
+    mod (abs n) 10
```

Comment:
If `div` is equal to 0, that would mean that mod of the number would be equal to the number itself, so no need to do `mod` in here:

```suggestion
    abs n
```

Reply:
Thanks! Makes sense